### DISCUSSION TITLE
Wishlist

### DISCUSSION THREAD

**Discussion by yl-1993** [openxrlab/xrmocap#9] [Ideas]
### Welcome notes

Please leave a comment on which new method/feature/dataset you would like us to add in XRMoCap.
The most upvoted suggestions will be prioritized!

**Comment by BingliangLi** [comment]
Please Support Human3.6M dataset, as many paper used it to train/evaluate :)

- Direct Multi-view Multi-person 3D Pose Estimation
- Multi-View Multi-Person 3D Pose Estimation with Plane Sweep Stereo
- Learnable Triangulation of Human Pose
- ...

Thanks for all the great work!
